tests/test_f3_orphan_child_recovery.py

```
@@ -0,0 +1,1270 @@
+            ),
+            backup_gateway=None,
+            lifecycle_gateway=None,
+            provider_identity_reader=_provider_identity,
+            retention_days=90,
+        )
+        self.service.f3_runtime = restarted
+        self.runtime = restarted
+        return restarted
+
+    async def test_orphaned_children_converge_in_one_sweep(self):
+        task, declarations = await self._build_live_orphan()
```

Ruff (0.16.1)

[warning] Unpacked variable `task` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)